### PR TITLE
Updated Platforms Guide

### PR DESCRIPTION
**Pull request by affederaffe**
Everything is now located in the CustomPlatforms repository for simplicity (like CustomAvatars does), therefore the updated guide. You probably don't want to merge this until I create the release.

### CODE REVIEW

File: wiki/models/platforms-guide.md [hunk 1]
```diff
@@ -44,23 +44,42 @@ Enabling the step effect changes the rings spacing when the specified event is c
...
+by first selecting what component, then what action.
+There are more predefined events like the `Combo Reached Event`,
+`Text Event Filter` (useful for custom counters) or `Every Nth ComboFilter`.
 
 ### Spectogram
 ![Spectogram](~@images/models/platforms/Spectogram.png)
 
-The spectrogram script works like the track rings script and also requires a prefab or gameObject. This will get stretched
-and shrunk according to the sound of the game and the variables provided. (Haven't played with this either).
+The `Spectrogram` script works like the track rings script and also requires a prefab.
```
Comment (namaki-mono) [inline wiki/models/platforms-guide.md:66]: Spectogram on line 63 vs. Spectrogram on line 66, unsure of which one is the correct spelling, but the one on line 66 has an 'r'.

File: wiki/models/platforms-guide.md [hunk 2]
```diff
@@ -44,23 +44,42 @@ Enabling the step effect changes the rings spacing when the specified event is c
...
+The colors will change according to the specified light ID.
 
 ### Song Events
 ![Song Event Handler](~@images/models/platforms/SongEventHandler.png)
 
-The event manager is the most useful script. With it you can trigger an action on any beat saber event (even unused ones).
-For adding an event press the `+` button underneath `On Trigger ()`. Drag in the object you want to manipulate into the box
-that just got created. Press the dropdown menu to the right and choose what that object should do, by first selecting what
-component, then what action. Make sure that you only use 1 event Handler per gameObject, as only 1 will work per gameObject.
+The `Event Manager` is the most useful script. With it you can trigger an action on any beat saber event (even unused).
```
Comment (namaki-mono) [inline wiki/models/platforms-guide.md:55]: Nitpick: "beat saber" should be capitalized as "Beat Saber" for consistency with the rest of the page.

File: wiki/models/platforms-guide.md [hunk 3]
```diff
@@ -44,23 +44,42 @@ Enabling the step effect changes the rings spacing when the specified event is c
...
 ### Spectogram
 ![Spectogram](~@images/models/platforms/Spectogram.png)
 
-The spectrogram script works like the track rings script and also requires a prefab or gameObject. This will get stretched
-and shrunk according to the sound of the game and the variables provided. (Haven't played with this either).
+The `Spectrogram` script works like the track rings script and also requires a prefab.
+This will get stretched and shrunk according to the sound of the game and the variables provided
+
+### Rotation Effetcs
+![Rotation Effetcs](~@images/models/platforms/RotationEffects.png)
```
Comment (namaki-mono) [inline wiki/models/platforms-guide.md:70]: "Rotation Effetcs" on line 69 and 70, should be "Rotation Effects".

File: wiki/models/platforms-guide.md [hunk 4]
```diff
@@ -44,23 +44,42 @@ Enabling the step effect changes the rings spacing when the specified event is c
...
+Both the `RotationEventEffect` and the `PairRotationEventEffec`t rotate your GameObjects based on the assigned song event.
```
Comment (namaki-mono) [inline wiki/models/platforms-guide.md:71]: The final character of `PairRotationEventEffect` is outside of the formatting block.

File: wiki/models/platforms-guide.md [hunk 5]
```diff
@@ -44,23 +44,42 @@ Enabling the step effect changes the rings spacing when the specified event is c
...
+There are more predefined events like the `Combo Reached Event`,
+`Text Event Filter` (useful for custom counters) or `Every Nth ComboFilter`.
 
 ### Spectogram
 ![Spectogram](~@images/models/platforms/Spectogram.png)
 
-The spectrogram script works like the track rings script and also requires a prefab or gameObject. This will get stretched
-and shrunk according to the sound of the game and the variables provided. (Haven't played with this either).
+The `Spectrogram` script works like the track rings script and also requires a prefab.
+This will get stretched and shrunk according to the sound of the game and the variables provided
```
Comment (namaki-mono) [inline wiki/models/platforms-guide.md:67]: Nitpick: Period missing at the end of this sentence.

File: wiki/models/platforms-guide.md [hunk 6]
```diff
@@ -44,23 +44,42 @@ Enabling the step effect changes the rings spacing when the specified event is c
...
 
 ### Song Events
 ![Song Event Handler](~@images/models/platforms/SongEventHandler.png)
 
-The event manager is the most useful script. With it you can trigger an action on any beat saber event (even unused ones).
-For adding an event press the `+` button underneath `On Trigger ()`. Drag in the object you want to manipulate into the box
-that just got created. Press the dropdown menu to the right and choose what that object should do, by first selecting what
-component, then what action. Make sure that you only use 1 event Handler per gameObject, as only 1 will work per gameObject.
+The `Event Manager` is the most useful script. With it you can trigger an action on any beat saber event (even unused).
+For adding an event press the + button underneath On Trigger ().
```
Comment (namaki-mono) [inline wiki/models/platforms-guide.md:56]: Maybe: The `+` button and `On Trigger ()` terms lost their block formatting, unsure if this was intentional.

File: wiki/models/platforms-guide.md [hunk 7]
```diff
@@ -44,23 +44,42 @@ Enabling the step effect changes the rings spacing when the specified event is c
...
+Those scripts are useful to create rotating laser (pairs) like in default environments.
+
+### Color Material
+This script lets you grab the current light, saber or obstacle color and pass it to a shader property.
+Use this if you want to create custom shaders.
+
+::: tip NOTE
+You don't need to restart your game anymore in order for changes to take effect.
+When you remove, create or change a file in your CustomPlatforms directory,
+these actions are reflected in-game and the corrosponding platform is deleted, loaded or updated.
```
Comment (namaki-mono) [inline wiki/models/platforms-guide.md:81]: corrosponding -> corresponding

**Review by namaki-mono — CHANGES_REQUESTED** [review]
Hello! I noticed a few minor issues (e.g. spelling, formatting, etc.) that should be easy to fix.

**Review by bloodcloak — APPROVED** [review]

**Pull request merged**